src/components/TodoContainer.js

```diff
@@ -22,21 +22,63 @@ const textFieldStyle = {
 };
 
 class TodoContainer extends Component {
+  constructor() {
+    super();
+    this.state = {
+      textField: '',
+      todoText: [],
+    };
+
+    this.handleKeyPress = this.handleKeyPress.bind(this);
+    this.handleChange = this.handleChange.bind(this);
+    this.deleteFunction = this.deleteFunction.bind(this);
+  }
+
+  handleKeyPress(evt) {
```

Comment:
you can also use de es6 arrow function sintax `handleKeyPress = (evt) => { ... }`